htmd/parameterization/cli.py

```diff
@@ -614,11 +607,15 @@ def main_parameterize(arguments=None):
         # Detect chiral centers
         initial_chiral_centers = detectChiralCenters(mol)
 
+        mm_minimizer = None
+        if args.min_type == 'mm' or args.dihed_opt_type == 'mm':
```

Comment:
Why do you need the second condition? Isn't this enough:
```python
if args.min_type == 'mm':
```

Reply:
No of course not. You could QM minimize the initial conformer and MM minimize the dihedrals

Reply:
`min_type` is the initial minimization of the structure. `dihed_opt_type` is the dihedral scan minimization. In either of these cases you need a mm minimizer. Hope this explains it a bit clearer.

Reply:
Aaaaah ok I see what you mean. Yes now that it's moved to the other section it makes sense

Reply:
In the past it was only instantiated once. Now of course since it's reinstantiated I can change the ifs